builtin/providers/docker/resource_docker_image_test.go

```diff
@@ -131,6 +131,6 @@ data "docker_registry_image" "foobarbaz" {
 }
 resource "docker_image" "foobarbaz" {
 	name = "${data.docker_registry_image.foobarbaz.name}"
```

Comment:
I think it would be useful to have 2 tests - 1 for pull_trigger and 1 for pull_triggers